src/voter/mod.rs

```diff
@@ -1040,20 +1190,14 @@ impl<H, N, E: Environment<H, N>, CommitIn, CommitOut> Voter<H, N, E, CommitIn, C
 			self.env.clone(),
 		);
 
-		let background = BackgroundRound {
-			inner: Arc::new(Mutex::new(prospective_round)),
-			task: None,
-			finalized_number: N::zero(), // TODO: do that right.
-		};
-
 		// TODO: should we clear `past_rounds`?
```

Comment:
for sure need to at some point, no? Just for memory usage reasons...

Maybe put this into a ticket?

Reply:
`past_rounds` is already slowly cleared as new blocks are finalized and old rounds become irrelevant